docs/validate_shcd.md

```diff
@@ -47,6 +47,34 @@ The tabs on the SHCD file to check, e.g. 10G_25G_40G_100G,NMN,HMN.
 The corners on each tab, comma separated e.g. ‘J37,U227,J15,T47,J20,U167’.
 
 
+### --out( <out>)
```

Comment:
this changeset is generated but there's something wrong w/ the generation.  repeated --out and --json for no reason.  hand edit this for now.